src/brs/BlockchainProcessorImpl.java

```diff
@@ -984,7 +985,7 @@ private void accept(Block block, Long remainingAmount, Long remainingFee)
     calculatedRemainingFee += atBlock.getTotalFees();
     // ATs
     if (subscriptionService.isEnabled()) {
-      calculatedRemainingFee += subscriptionService.applyUnconfirmed(block.getTimestamp());
+      calculatedRemainingFee += subscriptionService.applyUnconfirmed(block.getTimestamp(), block.getHeight());
```

Comment:
This and several similar are the adjustment for subscriptions to pay the FEE_QUANT as fee.